PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -20,6 +20,8 @@
         'keep *_slimmedMETs_*_*',
         'keep *_slimmedMETsNoHF_*_*',
         'keep *_slimmedMETsPuppi_*_*',
+        'keep *_slimmedMETsDeep_*_*',
+        'keep *_slimmedMETsDeepResp_*_*',
```

Comment:
all other ```slimmedMET*``` objects have some more subtle structure and come from ```PATMETSlimmer```.
There is nothing similarly "slimmed" in the DeepMET case.
I'm thinking ```deepMETs``` for the former.
The "Resp" apparently stands for response-corrected. Is it the particle-level response or some more general MET response? I'm thinking that a more descriptive (perhaps longer) name would be useful.
Considering that "slimmed" can be omitted here, the longer replacement of "Resp" may not change the total length of the product.

To fit the MET object framework, the "Resp" case may need to enter as a correction to the ```deepMETs``` (that's unless I'm overloading the meaning of "correction" here).

Reply:
> all other `slimmedMET*` objects have some more subtle structure and come from `PATMETSlimmer`.
> There is nothing similarly "slimmed" in the DeepMET case.
> I'm thinking `deepMETs` for the former.

Sounds good to me.

> The "Resp" apparently stands for response-corrected. Is it the particle-level response or some more general MET response? I'm thinking that a more descriptive (perhaps longer) name would be useful.
> Considering that "slimmed" can be omitted here, the longer replacement of "Resp" may not change the total length of the product.
> 
> To fit the MET object framework, the "Resp" case may need to enter as a correction to the `deepMETs` (that's unless I'm overloading the meaning of "correction" here).

"Response-corrected" is actually a bit of a misnomer: The two DeepMET versions have a different loss. The first version uses mean-squared-error loss, and has more optimal variance in the bias-variance trade-off and tends to be better for searches in MET resolution tails. The second version has an additional term in the loss yielding a better response and is such better-suited for analyses where unity response matters, in particular at low recoil. There is hence no correction applied. Would something like "deepMETsUnityResp" or "deepMETsUnityResponse" be more clear? (though this would technically only be true beyond some threshold)

Reply:
I see. So, Resp is not really a modifier of the former. Perhaps a more balanced naming is better than.
Still a bit offhand, how about ```deepMETsResponseTune``` and ```deepMETsResolutionTune``` or even  shorter ```deepMETsForResponse``` and ```deepMETsForResolution``` ... or ```deepMETsMinTails``` and ```deepMETsMinBias```?